server/src/api/modules/groups/group.types.ts

```diff
@@ -11,6 +11,7 @@ export interface GroupListItem extends Omit<Group, 'verificationHash'> {
 export interface GroupDetails extends GroupListItem {
   socialLinks?: GroupSocialLinks;
   memberships: MembershipWithPlayer[];
+  roleOrders?: GroupRoleOrder[];
```

Comment:
I wonder if from an API design prespective, it would be better to have this be either an empty array by default, or null by default (instead of non-existant/undefined)

@Jonxslays might have some thoughts

Reply:
Either of those would be better than undefined. I would lean towards the empty array.

Reply:
Agreed, and also `socialLinks` shouldn't be undefinedable

Reply:
From a non breaking change perspective it seems better to make it optional. Since you don't know like what type of serializes people are using to consume the endpoints? Maybe I'm overthinking it

Reply:
done